.github/workflows/tests.yml

```diff
@@ -153,11 +153,10 @@ jobs:
       matrix:
         python-version: ["3.8", "3.11"]
         java-version: ["17"]
-        gradle-version: ["7.3"]
-        ghidra-version: ["10.3"]
-        public-version: ["PUBLIC_20230510"] # for ghidra releases
-        jep-version: ["4.1.1"]
-        ghidrathon-version: ["3.0.0"]
+        ghidra-version: ["11.0.1"]
+        public-version: ["PUBLIC_20240130"] # for ghidra releases
+        jep-version: ["4.2.0"]
```

Comment:
This is no longer needed as `requirements.txt` used later handles installing the correct Jep version. Let’s remove this and the “Install Jep …” action that follows.